tests/phpunit/tests/rest-api/rest-controller.php

```diff
@@ -43,6 +43,10 @@ public function setUp() {
 						'type'   => 'string',
 						'format' => 'email',
 					),
+					'someuuid'    => array(
+						'type'   => 'string',
+						'format' => 'uuid',
+					)
```

Comment:
```suggestion
					),
```
We need a comma here to pass linting.